src/components/DataTable/index.tsx

```diff
@@ -55,6 +58,9 @@ export function DataTable<TData, TValue>({
   pagination: paginationConfig,
   rowSelection: rowSelectionConfig,
   initialRowSelection = {},
+  defaultColumnSize,
```

Comment:
@drkedug did it work? if so, showing it in one column of the stories' example is good.